repository/e.json

```diff
@@ -603,17 +603,12 @@
 			]
 		},
 		{
-			"name": "Elm Language Support",
-			"details": "https://github.com/elm-community/SublimeElmLanguageSupport",
-			"labels": ["language syntax"],
-			"previous_names": ["Elm Syntax Highlighting"],
+			"name": "Elm Format on Save",
+			"details": "https://github.com/evancz/elm-format-on-save",
+			"labels": ["elm", "elm-format", "formatting"],
```

Comment:
I doubt you'll find any other package that provides the `elm-format` tag, so it might as well be redundant, but won't hurt at least for now.

Reply:
That makes sense! I removed it in this commit: https://github.com/wbond/package_control_channel/pull/7653/commits/d57f36b87cf9f5e8a993ff2d0a811e56a3616dc0